metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/ELASTICSEARCH/2.3.3/configuration/elastic-site.xml

```diff
@@ -27,6 +27,14 @@
         <description>Cluster name identifies your cluster</description>
     </property>
     <property>
+        <name>masters_also_are_datanodes</name>
+        <value>"false"</value>
```

Comment:
Just looked at it again, it was a quick hack to overcome a silly bug in ES where False != false. Ambari does some tricks under the hood with bools where they come out as True/False causing ES to fail. I can struggle with it some more if it's important to you, but I think it's going to end with 

> if False then value = "false" else value = "true"

 somewhere in the code.